### PR TITLE
doc: move getting started guides to a section of their own

### PR DESCRIPTION
**Pull request by greg-fer**
Moved device getting started guides to evaluation guides above installation. NCSDK-22328.

----

This PR will be followed by a second and third PR that update Developing with guides and Working with sections, respectively.

### CODE REVIEW

**Review by OisinFitz — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

**Review by divipillai — APPROVED** [review]
As discussed offline with @greg-fer : Please note the whole section of "Building and debugging" of the nRF70 GSG will be moved to the developing with guide of the nRF70 Series, in a separate PR, as these require NCS installation and is part of the development process.

**Review by annwoj — APPROVED** [review]

**Comment by greg-fer** [comment]
Image size warning will be handled in https://nordicsemi.atlassian.net/browse/NCSDK-27322

**Review by umapraseeda — CHANGES_REQUESTED** [review]
Don't we need the new redirects for the gsg added in the redirects.py file?

**Comment by greg-fer** [comment]
> Don't we need the new redirects for the gsg added in the redirects.py file?

Updated.

File: doc/_utils/redirects.py [hunk 1]
```diff
@@ -217,12 +217,15 @@
     ("working_with_nrf/nrf52/features","device_guides/working_with_nrf/nrf52/features"),
     ("ug_nrf52_gs","device_guides/working_with_nrf/nrf52/gs"),
     ("working_with_nrf/nrf52/gs","device_guides/working_with_nrf/nrf52/gs"),
+    ("device_guides/working_with_nrf/nrf52/gs", "gsg_guides/nrf52_gs"),
     ("ug_nrf5340","device_guides/working_with_nrf/nrf53/nrf5340"),
     ("working_with_nrf/nrf53/nrf5340","device_guides/working_with_nrf/nrf53/nrf5340"),
+    ("device_guides/working_with_nrf/nrf53/nrf5340", "gsg_guides/nrf5340_gs"),
```
Comment (umapraseeda) [inline doc/_utils/redirects.py:223]:
```suggestion
    ("device_guides/working_with_nrf/nrf53/nrf5340_gs", "gsg_guides/nrf5340_gs"),
```

**Review by umapraseeda — APPROVED** [review]

**Comment by umapraseeda** [comment]
@gmarull, could we get this merged by ignoring the compliance error on image size? These are already existing images and were just moved around.

**Pull request merged**